server/zoom/meeting.go

```diff
@@ -3,25 +3,39 @@
 
 package zoom
 
+// MeetingType as defined at https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate
+type MeetingType int
+
+const (
+	// Instant meeting
+	Instant MeetingType = 1
+	// Scheduled meeting
+	Scheduled MeetingType = 2
+	// RecurringWithNoFixedTime meeting
+	RecurringWithNoFixedTime MeetingType = 3
+	// RecurringWithFixedTime meeting
+	RecurringWithFixedTime MeetingType = 8
```

Comment:
Can we have these prefixed with `MeetingType`? In other parts of the code, reading `Instant` is not easy to know that it is a meeting type